coordinator/persistence-api/src/main/java/org/apache/cassandra/stargate/exceptions/RequestFailureReason.java

```diff
@@ -20,13 +20,17 @@
 import static java.lang.Math.max;
 
 public enum RequestFailureReason {
+  // We have codes assigned for all reasons that exist in Apache Cassandra && HCD
```

Comment:
Should it say "DataStax HCD" as context for anyone outside DS? (HCD as acronym being quite new and not widely known outside)?

Reply:
Sure, I will update that